packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -707,9 +707,8 @@ export class ContainerRuntime extends EventEmitter
 
         const loadedFromSequenceNumber = this.deltaManager.initialSequenceNumber;
         const isSummarizerClient = this.clientDetails.type === summarizerClientType;
-        // Use runtimeOptions if provided, otherwise check localStorage, defaulting to true/enabled.
-        const enableSummarizerNode = this.runtimeOptions.enableSummarizerNode
-            ?? (typeof localStorage === "object" && localStorage?.fluidDisableSummarizerNode ? false : true);
+        // Use runtimeOptions if provided, otherwise default to true/enabled.
+        const enableSummarizerNode = this.runtimeOptions.enableSummarizerNode ?? true;
```

Comment:
fyi for future booleans, for user usage consistency usually, you want to default to false for these sort of settings. So in this case, the setting probably could have been this.runtimeOptions.disableSummarizerNode and this check would be simply "enableSummarizerNode = !this.runtimeOptions.disableSummarizerNode" or "disableSummarizerNode = this.runtimeOptions.disableSummarizerNode" :P

Reply:
Generally agree, that's why the localStorage was `fluidDisableSummarizerNode`. I decided to just be consistent with the rest of runtime options this time though, which were all `true` to enable.